static/js/views/game.js

```diff
@@ -239,21 +243,23 @@ export class GameUX {
             this.scrollboxBackground.parent.removeChild(this.scrollboxBackground)
             this.gameLogScrollbox = this.scrollbox()
         }
-        if (this.opponent(game)) {
-            let loadingImages = this.loadInPlayImages(game);
-            loadingImages = this.loadHandAndSelectionImages(game) || loadingImages;
-            loadingImages = this.loadStackSpellImages(game) || loadingImages;
-            loadingImages = this.loadCastingSpellImage(game, message) || loadingImages;
+        const currentGame = this.actionQueue[0].game
```

Comment:
is actionQueue guaranteed to be non-empty when you're trying to grab [0]?

Reply:
it is because refresh() gets called in 2 places.

On line 498 of Game.js:

            if (this.actionQueue.length) {
                this.refresh(this.actionQueue[0].game, this.actionQueue[0].message)
            } 
            
And on line 77/78 of GameRoom.js:

            this.gameUX.actionQueue.push({game, message});
            this.gameUX.refresh(game, message);